recipes/python-symengine/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
```

Comment:
That's confusing. I did merge it into the 1.21.x branch before tagging it. https://github.com/conda/conda-build/commit/63b4db5826bcc6bf77242fc4e53caa3afc27cf40 

I may have done it clumsily. I'm new to managing multiple release branches.